#Joint values for initial and final scanning configurations
scan_initial_posture: [0.0, 0.1, 1.4, 0.0, 0.0]
scan_final_posture: [0.0, 0.1, 2.6, 0.0, 0.0]

#Laser to end-effector transform
eMl: [-0.02, 0.0, 0.12, 0.0, 0.0, 0.0]

#Arm base to camera transform
bMc: [-0.2, -0.1, -0.13, 0.0 , 0.0, -1.5708]

